src/scss/components/_product-detail.scss

```diff
@@ -0,0 +1,188 @@
+.product-detail {
+  background-color: var(--gray);
+  padding-top: 0;
+  padding-bottom: 112px;
+
+  @include breakpoint(sm) {
+    padding-top: 52px;
+  }
+
+  .col-sm-6,
+  .col-sm-4 {
+    @include breakpoint(sm) {
+      padding: 0;
+    }
+  }
+
+  &__images {
+    display: flex;
+    width: 92vw;
```

Comment:
Try using a different `width` for the image in mobile, so that you can add a spacing between them. Something like:
`width: 78vw;` and add a `margin-right` to the images.